util/wait_group_wrapper_test.go

```diff
@@ -56,6 +57,7 @@ func TestNotifyErrorWaitGroupWrapper(t *testing.T) {
 	var wg = NewNotifyErrorWaitGroupWrapper(errCh)
```

Comment:
This abstraction is poor designed, error prone, and make the use get stuck easily!
I'd suggest just remove it.

Reply:
I think so. it is used in the analysis executor. so I think it may be suitable for their use.